Earthfile

```diff
@@ -389,7 +389,7 @@ dind:
 dind-alpine:
     FROM docker:dind
     RUN apk add --update --no-cache docker-compose
-    ARG EARTHLY_TARGET_TAG_DOCKER
+    ARG --required EARTHLY_TARGET_TAG_DOCKER
```

Comment:
Isn't this a builtin arg?

Reply:
ah right! I wonder if this should cause an error in this case?

Reply:
Yeah, it might be a good idea

Reply:
I created https://github.com/earthly/earthly/issues/1625 to track this.